packages/api/src/derives/staking/stakingAccount.ts

```diff
@@ -60,7 +48,7 @@ export function queryStakingAccountInfo(
 
       return retrieveStakingAccountDetails(api, stashId).pipe(
         switchMap(
-          ([controllerIdOpt, nominatorsOpt, rewardDestination, stakers, [validatorPrefs], nextKeys]): Observable<
+          ([controllerIdOpt, nominatorsOpt, rewardDestination, stakers, [validatorPrefs]]): Observable<
```

Comment:
Can you please explain what is this change for?

Reply:
the nextKeys does not exist.. 
The query for four fields  -  [api.query.staking.bonded, stashId],
    [api.query.staking.nominators, stashId],
    [api.query.staking.payee, stashId],
    [api.query.staking.stakers, stashId],
    [api.query.staking.validators, stashId], are mapped to 
[controllerIdOpt, nominatorsOpt, rewardDestination, stakers, [validatorPrefs]]

Reply:
The question is would it make anything fail? If so, shouldn't we add a test to catch such an issue later?